### PR TITLE
Global icons

### PR DESCRIPTION
**Pull request by huckocs**
This adds the global icons provided by James Alegria to the global-context. It is purely a folder of svg images, no code is provided

### CODE REVIEW

File: packages/global-context/HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # History
 
+## 2.7.0 (2019-07-04)
```
Comment (davidpauljunior) [inline packages/global-context/HISTORY.md]:
```suggestion
## 2.7.0 (2019-07-26)
```

File: packages/global-context/README.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ As well as the SCSS helpers documented here, we also have a collection of [JavaS
 	- [Color](#color)
 	- [Foreground color](#foreground-color)
 	- [Layers](#layers)
+- [Icons](#inons)
```
Comment (davidpauljunior) [inline packages/global-context/README.md]:
```suggestion
- [Icons](#icons)
```

File: packages/global-context/README.md [hunk 3]
```diff
@@ -166,6 +167,17 @@ z-index: z("layer-name", $my-map); //return z-index from $my-map map
 z-index: z("layer-name", "layer-variant", $my-map); //return z-index a name and variant from $my-map map
 ```
 
+## Icons
+A set of icons is provided for use across all brands in the /img/icons folder. Implementation is currently done at the product level due to the varying ways SVG Icons can be implemented. There is no agreed approach to this currently in Springer Nature.
```
Comment (davidpauljunior) [inline packages/global-context/README.md]: Maybe link `img/icons folder` to the folder?

File: packages/global-context/README.md [hunk 4]
```diff
@@ -166,6 +167,17 @@ z-index: z("layer-name", $my-map); //return z-index from $my-map map
 z-index: z("layer-name", "layer-variant", $my-map); //return z-index a name and variant from $my-map map
 ```
 
+## Icons
+A set of icons is provided for use across all brands in the /img/icons folder. Implementation is currently done at the product level due to the varying ways SVG Icons can be implemented. There is no agreed approach to this currently in Springer Nature. 
+
+There is a recommendation that SVG Icons are inlined, but this ia a choice.
```
Comment (davidpauljunior) [inline packages/global-context/README.md]:
```suggestion
There is a recommendation that SVG Icons are inlined, but this is a choice.
```

File: packages/global-context/README.md [hunk 5]
```diff
@@ -166,6 +167,17 @@ z-index: z("layer-name", $my-map); //return z-index from $my-map map
...
+
+For further information on the various implemenation methods, please read this article https://css-tricks.com/pretty-good-svg-icon-system/
```
Comment (davidpauljunior) [inline packages/global-context/README.md]:
```suggestion
For further information on the various implementation methods, please read this article https://css-tricks.com/pretty-good-svg-icon-system/
```

File: packages/global-context/README.md (hunk 3 above)
Comment (huckocs) [reply packages/global-context/README.md]: Love to, how?

Comment (davidpauljunior) [inline packages/global-context/README.md]:
```suggestion
A [set of icons](img/icons) is provided for use across all brands. Implementation is currently done at the product level due to the varying ways SVG Icons can be implemented. There is no agreed approach to this currently in Springer Nature. 
```

Comment (davidpauljunior) [reply packages/global-context/README.md]: I think the path is right in my suggest.  Can check once it's added in.

File: packages/global-context/README.md [hunk 6]
```diff
@@ -166,6 +167,17 @@ z-index: z("layer-name", $my-map); //return z-index from $my-map map
...
 ```
 
+## Icons
+A [set of icons](img/icons) is provided for use across all brands. Implementation is currently done at the product level due to the varying ways SVG Icons can be implemented. There is no agreed approach to this currently in Springer Nature. 
+
+There is a recommendation that SVG Icons are inlined, but this is a choice.
+
+For further information on the various implementation methods, please read this article https://css-tricks.com/pretty-good-svg-icon-system/
+
+The simplest way is to just inline them, as detailed in this article https://css-tricks.com/pretty-good-svg-icon-system/
```
Comment (davidpauljunior) [inline packages/global-context/README.md]:
```suggestion
```

File: packages/global-context/README.md [hunk 7]
```diff
@@ -166,6 +167,17 @@ z-index: z("layer-name", $my-map); //return z-index from $my-map map
 z-index: z("layer-name", "layer-variant", $my-map); //return z-index a name and variant from $my-map map
 ```
 
+## Icons
+A [set of icons](img/icons) is provided for use across all brands. Implementation is currently done at the product level due to the varying ways SVG Icons can be implemented. There is no agreed approach to this currently in Springer Nature. 
+
+There is a recommendation that SVG Icons are inlined, but this is a choice.
```
Comment (davidpauljunior) [inline packages/global-context/README.md]:
```suggestion
There is a recommendation that [SVG sprites](https://css-tricks.com/svg-sprites-use-better-icon-fonts/) are used, but this is a choice. The sprite can be made manually or if you use Gulp then they can be made automatically with [SVG Store](https://www.npmjs.com/package/gulp-svgstore).
```

**Pull request merged**